src/wasm/kotlin/wasm.arcs

```diff
@@ -26,7 +26,7 @@ resource InfoResource
 store InfoStore of [Info] in InfoResource
 
 
-particle TestParticle in 'build/bin/wasm32/mainDebugExecutable/main.wasm'
+particle TestParticle in '../../../bazel-bin/src/wasm/kotlin/test_harness.wasm'
```

Comment:
I feel like we should do something better than this one day, but I don't have any ideas right now... :-/

Reply:
I've been working on this in another PR. 

Ideally, I'd like to get hot code reloading working with bazel. Then we can use HCR and ibazel builds at once for max dev efficiency. 

I'll try to get to a clean solution.

Reply:
Sounds like a great idea! :-)

Reply:
Note that using the name 'build' is non-ideal, because people on case-insensitive file systems on Windows and OSX will end up with a conflict between bazel's BUILD file, and a build directory relative to the BUILD file with the same name.